src/ui/qgsuserprofileoptionswidgetbase.ui

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>QgsUserProfileOptionsWidgetBase</class>
+ <widget class="QWidget" name="QgsUserProfileOptionsWidgetBase">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>462</width>
+    <height>378</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>User Profile Settings</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout_2">
+   <item>
+    <widget class="QGroupBox" name="mProfilePolicyGroupBox">
+     <property name="title">
+      <string>Which profile should be used when QGIS starts ?</string>
+     </property>
+     <layout class="QGridLayout" name="gridLayout">
+      <item row="5" column="0" colspan="2">
+       <widget class="QLabel" name="label">
+        <property name="font">
+         <font>
+          <pointsize>9</pointsize>
+          <italic>true</italic>
+         </font>
+        </property>
+        <property name="text">
+         <string>These settings do not apply if a profile was explicitly selected with the --profile command line argument</string>
+        </property>
+        <property name="textFormat">
+         <enum>Qt::PlainText</enum>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="2" column="0" colspan="2">
+       <widget class="QRadioButton" name="mAskUser">
+        <property name="text">
+         <string>Let user choose profile at start up</string>
+        </property>
+       </widget>
+      </item>
+      <item row="1" column="0">
+       <widget class="QRadioButton" name="mDefaultProfile">
+        <property name="text">
+         <string>Always use profile</string>
+        </property>
+       </widget>
+      </item>
+      <item row="1" column="1">
+       <widget class="QComboBox" name="mDefaultProfileComboBox">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0" colspan="2">
+       <widget class="QRadioButton" name="mLastProfile">
+        <property name="text">
+         <string>Use last closed profile</string>
+        </property>
+       </widget>
+      </item>
+     </layout>
+    </widget>
+   </item>
+   <item>
+    <widget class="QGroupBox" name="mProfileSelectorGroupBox">
+     <property name="title">
+      <string>Profile Selector</string>
+     </property>
+     <layout class="QFormLayout" name="formLayout">
+      <item row="0" column="0" colspan="2">
+       <widget class="QCheckBox" name="mAllowProfileCreation">
+        <property name="text">
+         <string>Let user create a new profile when starting QGIS</string>
```

Comment:
This is a bit of a strange option to me -- either a user is in control of their local setup and can create new profiles as wanted (or ignore that ability), OR they aren't and their QGIS environment is being locked down for some reason... in which case this setting shouldn't be exposed as a user controlled option.

Reply:
Yeah, I wasn't too sure about it either... I will just drop it.